messaging-experiments-82.yaml

```diff
@@ -233,6 +233,76 @@
   targeting: browserSettings.update.channel == 'release' &&
     localeLanguageCode == 'en' && region == 'US' &&
     'app.shield.optoutstudies.enabled'|preferenceValue
+
+- id: WIZARD-LESS-PASSWORD-IMPORT-AUTOCOMPLETE-EXPERIMENT
+  enabled: true
+  targeting: firefoxVersion >= '82.' &&
+    browserSettings.update.channel != 'release' &&
+    localeLanguageCode == 'en' &&
+    (isFirstStartup ||
+     'password-autocomplete-wizardless' in activeExperiments)
```

Comment:
Looks like we need to remove people who opt out:
https://github.com/mozilla/messaging-system-inflight-assets/blob/79345e8aacef3f4c544489f720c9bcae9f1fff5c/messaging-experiments-82.yaml#L235